tooling/scripts/sort-poms.groovy

```diff
@@ -187,14 +188,16 @@ void sortModules(String[] sortModulesPaths) {
 
 void updateMvndRules(String[] updateMvndRuleDirs) {
     final Set<String> extensionArtifactIds = [] as TreeSet
-    Files.list(baseDir.resolve('extensions'))
+    Stream.of('extensions', 'extensions-core')
```

Comment:
not very groovysh :)

Reply:
Yes, I am a proper Groovy noob! If there was a plugin to run Java as a script, I'd very much prefer writing it in Java :)